static/styles/zulip.css

```diff
@@ -2334,19 +2334,15 @@ button.topic_edit_cancel {
     display: none;
 }
 
-.empty_feed_notice h4 {
+.empty_feed_notice h4,
+.empty_feed_notice p {
     text-align: center;
 }
 
 .notification {
     cursor: pointer;
 }
 
-#empty_narrow_private_message p,
```

Comment:
what does this change do?

Reply:
@rishig I think this is to center align the `Learn more about ... documentation`. I have already made a pull request regarding this in [PR](https://github.com/zulip/zulip/pull/6197).

Reply:
Interesting! I believe it, but why does removing the `text-align: center` cause the centering to happen?
Also @synicalsyntax, probably makes sense to rebase off of #6197; sorry for not finding it earlier :/.

Reply:
@rishig Rebased